src/app/modules/pages/components/sign-up-form/sign-up-form/sign-up-form.component.ts

```diff
@@ -18,20 +18,22 @@ export class MyErrorStateMatcher implements ErrorStateMatcher {
 	styleUrls: ['./sign-up-form.component.scss'],
 })
 export class SignUpFormComponent {
-	public matcher = new MyErrorStateMatcher();
+	public visible = false;
+	public changeType = false;
+	public passwordErrorMatcher = new PasswordErrorStateMatcher();
+	public signUpForm = new FormGroup(
+		{
+			name: new FormControl('', Validators.required),
+			lastName: new FormControl('', Validators.required),
+			email: new FormControl('', [Validators.required, Validators.email]),
+			password: new FormControl('', Validators.required),
+			confirmPassword: new FormControl('', Validators.required),
+		},
+		{ validators: PasswordValidators.controlValueAreEqual('password', 'confirmPassword') }
+	);
 
-	public signUpForm = new FormGroup({
-		name: new FormControl('', Validators.required),
-		lastName: new FormControl('', Validators.required),
-		email: new FormControl('', [Validators.required, Validators.email]),
-		password: new FormControl('', Validators.required),
-		confirmPassword: new FormControl('', Validators.required),
-	});
-
-	private readonly notifier: NotifierService;
-
-	constructor(private httpClient: HttpClient, notifierService: NotifierService) {
-		this.notifier = notifierService;
+	constructor(private httpClient: HttpClient, private notifierService: NotifierService) {
+		this.notifierService = notifierService;
```

Comment:
This is what you are doing right now
<img width="1782" alt="image" src="https://user-images.githubusercontent.com/29478275/184503779-90653cdc-714b-4f15-9bc2-b739274e5e18.png">

Here is what you can do. As you can see in the example below private/protected/public property inside the class will be automatically added to this, so you don't need manually assign it.
<img width="1778" alt="image" src="https://user-images.githubusercontent.com/29478275/184503806-8e3eedc2-eeba-411b-a59d-4eabe746e57e.png">